### DISCUSSION TITLE
Setting up ElastAlert

### DISCUSSION THREAD

**Discussion by yong362** [Security-Onion-Solutions/securityonion#10402] [Unsupported Versions]
Hi I need some help with troubleshooting. 

I followed the guide on how to setup email to external using https://docs.securityonion.net/en/2.3/elastalert.html#email-internal
I created the rule using so-elastalert-create and I tested the rule using **so-elastalert-test**, it runs successfully but when I tried running it using **so-elastalert-test -a**, it shows multiple error. How can I resolve this issue? Below is the error message. Hope someone can help me. :) Thank you in advance!
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13671289/d214c154-97eb-40eb-8c87-b157808eb887)

**Comment by andre-dessert** [comment]
What is the actual output from so-elastalert-test?
